jekyll/_data/sidenav.yml

```diff
@@ -1301,25 +1250,80 @@ ja:
     - name: How-To Guides
       i18n_name: ガイド
       children:
-        - name: Using matrix jobs
-          link: ja/using-matrix-jobs
-          i18n_name: マトリックスジョブ
         - name: Using branch filters
           link: ja/using-branch-filters
           i18n_name: ブランチフィルター
-        - name: Using dynamic configuration
-          link: ja/using-dynamic-configuration
-          i18n_name: ダイナミックコンフィグ
         - name: Selecting a workflow to run
           link: ja/selecting-a-workflow-to-run-using-pipeline-parameters
           i18n_name: 実行するワークフローの指定
         - name: Installing and using docker-compose
           link: ja/docker-compose
           i18n_name: docker-composeの使用
+        - name: Rename organizations and repositories
+          link: ja/rename-organizations-and-repositories
+          i18n_name: 組織名とリポジトリ名の変更
         - name: Migrate from deploy to run
           link: ja/migrate-from-deploy-to-run
           i18n_name: deployからrunへの移行
 
+- name: Optimizations
```

Comment:
@kbatuigas is there a reason we're not using the same section titles as the English here? (Also Security rather than Security and Permissions and maybe some others). I'm having trouble reviewing properly because I still can't build!

Reply:
@rosieyohannan ah, yes - in cases like these I just left it at the existing/"old" EN equivalent because I wasn't sure that the JA translation would be the exact same (Optimizations vs Optimize). I haven't requested actual translation work for this PR, but just a quick check that the section titles still look OK and mostly match up. Figured it would make sense to put in the localization request only when the reorg for EN is complete?